### PR TITLE
Show committer, upstream and date in branch finder

### PR DESCRIPTION
**Pull request by kyoh86**
Show short informations for all branches.

telescope-git

for: #569

### CODE REVIEW

File: lua/telescope/builtin/files.lua [hunk 1]
```diff
@@ -401,7 +401,11 @@ files.tags = function(opts)
 end
 
 files.current_buffer_tags = function(opts)
-  return files.tags(vim.tbl_extend("force", {prompt_title = 'Current Buffer Tags', only_current_file = true, hide_filename = true}, opts))
+  return files.tags(vim.tbl_extend("force", {
```
Comment (kyoh86) [inline lua/telescope/builtin/files.lua:404]: out of context: this line was reported by linter, so I fixed it.

File: lua/telescope/pickers/highlights.lua [hunk 2]
```diff
@@ -43,7 +43,7 @@ end
 function Highlighter:clear_display()
   if not self
       or not self.picker
-      or not self.picker.results_bufnr 
+      or not self.picker.results_bufnr
```
Comment (kyoh86) [inline lua/telescope/pickers/highlights.lua:46]: out of context: this line was reported by linter, so I fixed it.

**Comment by tami5** [comment]
That looks really nice, I'll test it out in few hours

**Comment by tami5** [comment]
LGTM,

File: lua/telescope/builtin/git.lua [hunk 3]
```diff
@@ -77,32 +78,81 @@ git.bcommits = function(opts)
...
   for _, v in ipairs(output) do
-    if not string.match(v, 'HEAD') and v ~= '' then
-      if vim.startswith(v, '*') then
-        table.insert(results, 1, v)
-      else
-        table.insert(results, v)
-      end
+    local entry = vim.fn.json_decode(v)
+    if entry.head then
+      goto continue
```
Comment (Conni2461) [inline lua/telescope/builtin/git.lua:114]: Can we please not use `goto`? It breaks lua 5.1 and some people have to build neovim with 5.1 
Example: https://github.com/nvim-telescope/telescope.nvim/issues/518

**Comment by Conni2461** [comment]
Also i think its a regression that the current branch is no longer indicated, don't you think so as well @ThePrimeagen, you requested that it is indicated in the first place. Now its no longer present at all.

Also with the date information we can also think of initial sorting based on the date.

**Pull request merged**